EIPS/eip-777.md

```diff
@@ -0,0 +1,545 @@
+---
+eip: 777
+title: A New Advanced Token Standard
+author: Jordi Baylina @jbaylina, Jacques Dafflon @jacquesd, Thomas Shababi @tshabs
```

Comment:
Please put usernames in parens, so they'll be automatically linkified and the bot can determine they're authors.